### PR TITLE
Add pterodactyl wings logs collection, parser & scenario

### PR DESCRIPTION
**Pull request by Lourys**
Pterodactyl is a free, open-source game server management panel.
Link to repository of Pterodactyl Wings program: https://github.com/pterodactyl/wings

### CODE REVIEW

**Comment by buixor** [comment]
Hello @Lourys !

Thanks for the PR. We'll have a look and let you know or merge, thanks :+1:

File: scenarios/lourys/pterodactyl-wings-bf.yaml [hunk 1]
```diff
@@ -0,0 +1,50 @@
+####################
+## Invalid format ##
+####################
+type: leaky
+name: lourys/pterodactyl-wings-bf
+description: "Detect invalid_format ssh bruteforce"
+filter: "evt.Meta.log_type == 'invalid_format'"
+leakspeed: "15s"
+references:
```
Comment (buixor) [inline scenarios/lourys/pterodactyl-wings-bf.yaml:9]: can you remove the reference ? it was present in ssh to showcase the possiblity

Comment (Lourys) [reply scenarios/lourys/pterodactyl-wings-bf.yaml]: Oh, okay!

**Comment by Lourys** [comment]
> Thanks for the PR, a few remarks before we can move and merge this :)
> 
> * can you remove the leftover Wikipedia link in the scenario ?
> * can you set a `log_type` more specific to pterodactyl ? ie. `pterodactly_invalid_format`

I just pushed the changes you requested! 😃

**Review by buixor — APPROVED** [review]
lgtm, thanks!

**Comment by buixor** [comment]
@Lourys : out of curiosity, do you know if there are other ways to login than `sftp` ? (it's not a blocker for merging, it's just to know if we should plan to extend it later)

**Review by blotus — APPROVED** [review]

**Comment by Lourys** [comment]
> @Lourys : out of curiosity, do you know if there are other ways to login than `sftp` ? (it's not a blocker for merging, it's just to know if we should plan to extend it later)

Yeah indeed, there are other way to login but the auth part is not managed by the "wings" program itself and/or the log events are not written in the log file(s).
I'll extend it if I found other use cases as I heavily use the Pterodactyl software. You can also push community members requests about pterodactyl to me if there are any!

Here's my discord tag if you want to contact: **Lourys#0359** (I'm on the Crowdsec discord server)

**Pull request merged**